tests/test_classification.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+from pyriemann_qiskit.classification import (QuanticSVM, QuanticVQC)
+
+
+def test_Quantic_init():
+    """Test init of quantum classifiers"""
+    # if "classical" computation enable,
+    # no provider and backend should be defined
+    q = QuanticSVM(target=1, quantum=False)
+    assert(not q.quantum)
+    assert(not hasattr(q, "backend"))
+    assert(not hasattr(q, "provider"))
+    # if "quantum" computation enabled, but no accountToken are provided,
+    # then "quantum" simulation will be enabled
+    # i.e., no remote quantum provider will be defined
+    q = QuanticSVM(target=1, quantum=True)
+    assert(q.quantum)
+    assert(hasattr(q, "backend"))
+    assert(not hasattr(q, "provider"))
+    # if "quantum" computation enabled, and accountToken is provided,
+    # then real quantum backend is used
+    # this should raise a error as uncorrect API Token is passed
+    try:
+        q = QuanticSVM(target=1, quantum=True, qAccountToken="Test")
+        assert(False)  # Should never reach this line
+    except Exception:
+        pass
+
+
+def test_Quantic_splitTargetAndNonTarget(get_covmats):
+    """Test splitTargetAndNonTarget method of quantum classifiers"""
+    covset = get_covmats(100, 3)
```

Comment:
Have a look to new tests: numeric parameters are defined in a separate line.
For eg, `n_matrices, n_channels = 100, 3`,
allowing non hardcoded asserts, like: `assert len(xta) == n_matrices//2`